### PR TITLE
Update sprints.md

### PR DESCRIPTION
**Pull request by rodrigogiraoserrao**

### CODE REVIEW

**Comment by rodrigogiraoserrao** [comment]
> hey @rodrigogiraoserrao would you mind actually adding the requirements for open source you mentioned on the page as well?

Of course; is this what you had in mind?

File: data/pages-content/sprints.md [hunk 1]
```diff
@@ -63,10 +63,11 @@ As per our tradition, the conference organisers will provide the rooms and facil
 
 If you are planning to run a sprint at EuroPython 2023, please add it at the bottom of this page [creating a pull request to our website](https://github.com/EuroPython/website/blob/main/data/pages-content/sprints.md):
 
-1. Copy the template at the bottom of the sprint list.
-2. Fill the template with your details.
-3. Add your sprint at the end of the list.
-4. Update the room occupancy in the list above.
+1. Make sure your sprint is about an _open source_ project.
```
Comment (Vaibhavs10) [inline data/pages-content/sprints.md:66]: Maybe also add a line about being mindful of how many people can they accommodate.. People who are not as aware of the project require way more hand-holding.

**Review by Vaibhavs10 — APPROVED** [review]

File: data/pages-content/sprints.md (hunk 1 above)
Comment (hypha) [inline data/pages-content/sprints.md:66]:
```suggestion
1. Make sure your sprint is an _open source_ project, with a link to a repo open to the public. 
```

**Comment by rodrigogiraoserrao** [comment]
@hypha @Vaibhavs10 what do you think of the new list?

**Comment by hypha** [comment]
@rodrigogiraoserrao happy for this to be merged?

**Comment by rodrigogiraoserrao** [comment]
> @rodrigogiraoserrao happy for this to be merged?

Yeah, absolutely. I don't have permission to merge; I'm not sure if you were waiting for me..?

**Pull request merged**